table/meta_blocks.cc

```diff
@@ -89,6 +89,7 @@ void PropertyBlockBuilder::AddTableProperty(const TableProperties& props) {
   Add(TablePropertiesNames::kColumnFamilyId, props.column_family_id);
   Add(TablePropertiesNames::kCreationTime, props.creation_time);
   Add(TablePropertiesNames::kOldestKeyTime, props.oldest_key_time);
+  Add(TablePropertiesNames::kFileCreationTime, props.file_creation_time);
```

Comment:
I think we shouldn't add it if the value is 0. This is a confusing value and also waste several bytes memory per SST file.

Reply:
done.